File: 181 excersises/174.py
def calcular_area(base, altura):
    return (base * altura) / 2

base = float(input("Ingresa la base del triangulo: "))
altura = float(input("Ingresa la altura del triangulo: "))

area = calcular_area(base, altura)
print(f"El area del triangulo es: {area}")

File: 181 excersises/174.cpp
#include <iostream>
using namespace std;

double calcularArea(double* base, double* altura) {
    return (*base * *altura) / 2.0;
}

int main() {
    double base, altura;
    
    cout << "Ingresa la base del triangulo: ";
    cin >> base;
    
    cout << "Ingresa la altura del triangulo: ";
    cin >> altura;
    
    double area = calcularArea(&base, &altura);
    
    cout << "El area del triangulo es: " << area << endl;
    
    return 0;
}